twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -41,7 +38,7 @@ class TwistlockCheck(AgentCheck):
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
 
-        self.last_run = datetime.utcnow()
+        self.last_run = datetime.now(tz.tzutc())
```

Comment:
whats the difference?

Reply:
`datetime.utcnow()` is returns an offset-naive datetime while `datetime.now(tz.tzutc())` is offset-aware

Datetime comparison [here](https://github.com/DataDog/integrations-core/blob/5f925f0dcd8c6c1f00ded34d89d820bf89784766/twistlock/datadog_checks/twistlock/twistlock.py#L89-L92) returns an error `TypeError: can't compare offset-naive and offset-aware datetimes` since parser.isoparse()` also returns an offset-aware datetime.

Reply:
Ahh wait. `last_run` appears to be compared to `published_date` which should be offset-naive.  https://github.com/DataDog/integrations-core/blob/5f925f0dcd8c6c1f00ded34d89d820bf89784766/twistlock/datadog_checks/twistlock/twistlock.py#L234

I'll revert the change for `last_run` but will keep it for `current_date`